packages/wonder-stuff-server-google/src/__tests__/start-server.test.js

```diff
@@ -0,0 +1,256 @@
+// @flow
```

Comment:
This is the remainder of the original `startGateway` tests from `shared` in RG that wasn't ported in the previous PR.